lib/serviceDiscovery/engines/consul/index.js

```diff
@@ -24,7 +24,29 @@ class ConsulService extends Service {
 		super();
 
 		if (!options.interface) {
-			throw new Error('Missing "interface" option for consul module');
+			// find the first interface with a valid IPv4
+			let validIface;
+
+			const interfaces = os.networkInterfaces();
+			for (let iface in interfaces) {
+				if (!interfaces.hasOwnProperty(iface)) {
```

Comment:
MAGE blew up in logging because more and more objects that Node returns don't have a prototype, and thus don't have `.hasOwnProperty`. You may wanna use `Object.keys()` and iterate with a regular for-loop.